src/components/Header/UserMenu.tsx

```diff
@@ -0,0 +1,78 @@
+import { Fragment } from 'react';
+import { HiUser } from 'react-icons/hi';
+import { Menu, Transition } from '@headlessui/react';
+import { type User } from '@supabase/gotrue-js/src/lib/types';
+import cn from 'classnames';
+
+import { useSignOut } from './hooks';
+
+export type UserMenuPros = {
```

Comment:
typo
```suggestion
export type UserMenuProps = {
```
also using interfaces for object types would be better